docker-prod-template.yml

```diff
@@ -0,0 +1,55 @@
+version: '2'
+services:
+  api:
+    image: openeats/api
+    command: ./base/gunicorn_start.sh
+    restart: on-failure
+    volumes:
+      - static-files:/code/static-files
+      - site-media:/code/site-media
+    depends_on:
+      - db
+    environment:
```

Comment:
The api needs to be aware of the following env vars as well:
- MYSQL_DATABASE
- MYSQL_ROOT_PASSWORD

The api needs them in order to login to the database and perform I/O